Feature/header

tipanazar opened the pull request.

tipanazar committed:
Merge pull request #13 from baza-trainee/develop

header branch update

tipanazar committed:
not formatted, done

tipanazar committed:
done

tipanazar committed:
upd

MaksVibe commented on cms/public/logo.svg:
when the design will be fixed, you should replace the svg with text, thanks.

MaksVibe commented on cms/src/components/Header/Header.styles.ts:
```diff
@@ -0,0 +1,87 @@
+  align-items: center;
+  border-radius: 4px;
+  width: 239px;
+  height: 56px;
```
make width responsive and remove height

MaksVibe commented on cms/src/components/Header/Header.styles.ts:
```diff
@@ -0,0 +1,87 @@
+  width: 100%;
+  color: #767676;
+  font-size: 16px;
+  background-color: #fcfcfc;
```
here will be bg transparent and for its parent <div> #fcfcfc

tipanazar committed:
fixed

Input: text and background color, adaptive width, height.
Logo: svg to text, paddings, added link to home page.

tipanazar committed:
Update Header.styles.ts

tipanazar committed:
Merge pull request #21 from baza-trainee/develop

Update from Develop

tipanazar committed:
Merge pull request #24 from baza-trainee/develop

Update from Develop

tipanazar committed:
header title styles change

tipanazar committed:
Header upd

tipanazar committed:
Update Header.tsx

tipanazar committed:
Update Header.styles.ts

MaksVibe committed:
fix: styles. feat: add layout container

MaksVibe committed:
Delete package-lock.json
